src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+:root {
+  --text-color: rgb(61, 78, 97);
+  --bg-gradi: linear-gradient(180deg, rgb(255, 255, 255), rgb(246, 246, 247));
+}
+
+body {
+  margin: 0;
+  font-weight: 300;
+  font-family: "Avenir Book", Helvetica, Arial, sans-serif;
+}
+
+::placeholder {
+  color: var(--text-color);
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 100%;
+}
+
+.search-bar__big {
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  width: 100%;
+  max-width: 900px;
+  padding: 25px 62px;
+  background: url("./images/Search.svg") 26px 25px no-repeat;
+}
+
+.search-bar__big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__big:focus {
+  border-radius: 3px;
+  border: 1px solid rgb(225, 231, 237);
+  background: url("./images/Search.svg") 26px 25px no-repeat, var(--bg-gradi);
+}
+
+.search-bar__big:focus::placeholder {
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 800;
+}
+
+.search-bar__small {
+  border: 1px solid #e1e7ed;
```

Comment:
you repeat a lot of values that are common for both - big and small

do the following:

```

.search-bar {
  border: 1px solid #e1e7ed;
  margin-top: 20px;
  width: 100%;
  max-width: 900px;
  padding: 25px 62px;
  background: url("./images/Search.svg") 26px 25px no-repeat;
}

.search-bar__small {
  set only values that not the same as in big input
}
```
in HTML for big input use `search-bar` class only, for small input - both `search-bar search-bar-small`